README.md

```
@@ -31,98 +31,80 @@
+- **Currently exploring**: Rust, Supabase, and advanced CI/CD with Codemagic.
+
+### ✍️ Latest from Medium
+<!-- BLOG-POST-LIST:START -->
+<!-- BLOG-POST-LIST:END -->
+
+### 🧰 Strategic Tech Stack
+<table>
+  <tr>
+    <td align="center" width="33%">
+      <b>Mobile Engineering</b><br><br>
+      <img src="https://skillicons.dev/icons?i=flutter,dart,kotlin,androidstudio,swift,apple&perline=3" />
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)